src/cli.c

```diff
@@ -722,6 +722,7 @@ static void usage(const char *cmd)
            "                       server. If omitted, the command runs as a client.\n"
            "  -e event-log-file    file to log events\n"
            "  -i interval          interval to reissue requests (in milliseconds)\n"
+           "  -I timeout           idle timeout (in milliseconds; default: 600,000)\n"
```

Comment:
```suggestion
           "  -t timeout           idle timeout (in milliseconds; default: 600,000)\n"
```

slightly better since -i is used for interval.

Reply:
The choice depends on if we would have other timeouts.

We currently use `-r` for initial RTO, we might introduce another timeout that is used to declare connection dead when there is no response from the peer.